### DISCUSSION TITLE
Borrar cuenta

### DISCUSSION THREAD

**Discussion by Cholo68** [Paco8/plugin.video.movistarplus#71] [Ideas]
Buenos dias, primero darte la enhorabuena Paco por este magnifico addon, funciona excelentemente bien.
Lo que echo de menos en el addon es que al igual que el de orange si tiene la opcion de eliminar cuenta en Movistar no hay esa opcion y la cuenta que quisiera eliminar no puedo hacerlo. No se si es que en el addon de Movistar no se puede implementar esa opcion o se te pasó por alto auque lo dudo. Lo dicho, muchas gracias por todo tu excelente trabajo

**Comment by kekolokocr7** [comment]
is dead the addons.

**Comment by Cholo68** [comment]
Eso no es cierto, puedes decir a lo mejor que a ti el addon no te funciona pero ya te aseguro yo que los addons de Paco funcionan perfectamente tanto Movistar, Orange y Skyshowtime.

**Comment by Paco8** [comment]
Es verdad, no tiene la opción de borrar cuenta. La puedes borrar con el explorador de ficheros del kodi, ve a carpeta del perfil, addon_data, plugin.video.movistarplus. Allí están todas las cuentas (account_1, account_2, etc.), borra la carpeta de la cuenta que quieres borrar.

**Reply by Cholo68** [reply]
SI, esa es la opcion que vi para poder eliminarla pero me extrañó que no tuviera esa opcion el addon por eso lo comenté. Muchas gracias
